### DISCUSSION TITLE
Readiness for .NET 8 Isolated

### DISCUSSION THREAD

**Discussion by ThatRendle** [Azure/azure-functions-dotnet-worker#1417] [Feedback]
Clearly the work has started to provide feature parity with in-process functions in terms of parameter binding, with the `Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs 5.1.0-preview1` package supporting `BlobClient` etc. as parameter types as per [the docs](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob?tabs=isolated-process%2Cextensionv5%2Cextensionv3&pivots=programming-language-csharp).

Is there a roadmap for delivering full SDK bindings across the other triggers, hopefully in time for .NET 8 GA in November? I see the [feature: SDK bindings](https://github.com/Azure/azure-functions-dotnet-worker/labels/feature%3A%20SDK%20bindings) label but it doesn't have a full set of issues, e.g. I can't see ServiceBus in there.

**Comment by fabiocav** [comment]
@markrendle apologies for the delayed response.

The epic issue [here](https://github.com/Azure/azure-functions-dotnet-worker/issues/1081) has been populated with the Service Bus items. Initial preview with partial features has been released and work for the additional functionality is in progress.

**Comment by fabiocav** [comment]
Closing this as the work is complete.
